lib/wampproto/message/util.rb

```diff
@@ -0,0 +1,476 @@
+# frozen_string_literal: true
+
+module Wampproto
+  # message module
+  module Message
+    # fields
+    class Fields
+      attr_accessor :request_id, :uri, :args, :kwargs, :session_id, :realm, :authid,
+                    :authrole, :authmethod, :authmethods, :authextra, :roles,
+                    :message_type, :signature, :reason, :topic, :extra,
+                    :options, :details, :subscription_id, :publication_id,
+                    :registration_id
+
+      def initialize # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
```

Comment:
@Mahad-10  you can remove the initializer method, as what is happening the initializer is the default. 
when you have `attr_accessor :attribute_name`
by default @attribute_name will be nil